apps/web-giddh/src/app/expenses/components/pending-list/pending-list.component.scss

```diff
@@ -62,7 +62,9 @@
 .td-search-box {
     position: relative;
 }
-
+.mat-column-s_no {
```

Comment:
change this classname and add proper class, according to our guideline @ravinderwalkover

Reply:
@aditi-tiwari17 this is dynamic class, can't control it.